### PR TITLE
`Typography` Website Doc Enhancement - Follow up

### PR DESCRIPTION
**Pull request by majedelass**
### :pushpin: Summary

If merged, this PR will remediate the failed tests on the previous Typography enhancements PR.

### :hammer_and_wrench: Detailed description

Fixing failed lint test and accessibility issues. Ensuring we pass all the tests now.

### :link: External links

Previous PR: [PR 2364](https://github.com/hashicorp/design-system/pull/2364)

***

### 👀 Component checklist

- [ ] Percy was checked for any visual regression

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: website/app/styles/pages/foundations/typography.scss [hunk 1]
```diff
@@ -4,7 +4,11 @@
  */
 #show-content-foundations-typography{
     .doc-typography-demo-display-list{
-        list-style-type: none;
         padding:0;
+        list-style-type: none;
+    }
+
+    .list-item-gap{
```
Comment (KristinLBradley) [inline website/app/styles/pages/foundations/typography.scss:11]:
```suggestion
    .list-item-gap {
```

File: website/app/styles/pages/foundations/typography.scss [hunk 2]
```diff
@@ -4,7 +4,11 @@
...
+        margin-bottom:24px;
```
Comment (KristinLBradley) [inline website/app/styles/pages/foundations/typography.scss:12]:
```suggestion
        margin-bottom: 24px;
```

File: website/app/styles/pages/foundations/typography.scss [hunk 3]
```diff
@@ -4,7 +4,11 @@
  */
 #show-content-foundations-typography{
```
Comment (KristinLBradley) [inline website/app/styles/pages/foundations/typography.scss:5]:
```suggestion
#show-content-foundations-typography {
```

File: website/app/styles/pages/foundations/typography.scss [hunk 4]
```diff
@@ -4,7 +4,11 @@
...
     .doc-typography-demo-display-list{
```
Comment (KristinLBradley) [inline website/app/styles/pages/foundations/typography.scss:6]:
```suggestion
    .doc-typography-demo-display-list {
```

File: website/docs/foundations/typography/partials/guidelines/guidelines.md [hunk 5]
```diff
@@ -97,18 +97,14 @@ Display styles should be used for headings or visual emphasis (like remaining ba
 Display text styles are available in the following sizes and weights:
 
 <ul class='doc-typography-demo-display-list'>
```
Comment (KristinLBradley) [inline website/docs/foundations/typography/partials/guidelines/guidelines.md:99]:
```suggestion
<ul class="doc-typography-demo-display-list">
```

File: website/app/styles/pages/foundations/typography.scss (hunk 1 above)
Comment (majedelass) [reply website/app/styles/pages/foundations/typography.scss:11]: To this point, I should have the class name reflect what this is doing more specifically to this: `typography-list-item-gap`. Making updates locally then passing it through.

File: website/docs/foundations/typography/partials/guidelines/guidelines.md [hunk 6]
```diff
@@ -121,12 +117,10 @@ Body text styles are available in the following weights and sizes:
...
```
Comment (KristinLBradley) [inline website/docs/foundations/typography/partials/guidelines/guidelines.md:117]:
```suggestion
<ul class="doc-typography-demo-display-list">
```

File: website/docs/foundations/typography/partials/guidelines/guidelines.md [hunk 7]
```diff
@@ -140,11 +134,9 @@ Code text styles come in the following weights and sizes:
 
 <ul class='doc-typography-demo-display-list'>
```
Comment (KristinLBradley) [inline website/docs/foundations/typography/partials/guidelines/guidelines.md:135]:
```suggestion
<ul class="doc-typography-demo-display-list">
```

File: website/app/styles/pages/foundations/typography.scss [hunk 8]
```diff
@@ -4,7 +4,11 @@
...
-        list-style-type: none;
         padding:0;
```
Comment (KristinLBradley) [inline website/app/styles/pages/foundations/typography.scss:7]:
```suggestion
        padding: 0;
```

**Review by KristinLBradley — APPROVED** [review]

**Review by KristinLBradley — APPROVED** [review]

**Pull request merged**